### DISCUSSION TITLE
Can this be used for Espn Baseball ?

### DISCUSSION THREAD

**Discussion by tomrulz** [cwendt94/espn-api#194] [ESPN API]
Let me start off by letting you guys know I am complete noob. However I am making progress. I see in the wiki it says it currently supports Football/Basketball but I wanted to know if it can support baseball? 

I got my SWID and ESPN_S2 and got the API installed using PIP. 

I am looking to capture each teams roster at 00:01 each Monday AM and the recent activity for the previous week each sunday at 11:59pm in my fantasy baseball league. 

I am going to try and give the API a shot at doing this, but I am not a programmer.

Thanks in advance for any help!!

**Comment by cwendt94** [comment]
Great question @tomrulz! Currently this package does not support baseball however I don't think it will take a lot of time to get the initial implementation setup.

Most of the code from Football/Basketball can be copied to get some of the basic features like Teams, Roster, Recent Activity. I will try to get it done over the weekend.

After that it wouldn't be too hard to use the API for what you want to do!

**Reply by tomrulz** [reply]
That would be super awesome. 

I am running a league where I need to keep track of each teams roster weekly for salary purposes and would love to have a way to setup an automatic process to pull Rosters and Recent Activity Log at set times. 

Please let me know if I can assist in any way, or If I can buy you a beer. 

Thanks, 
Tom!!!

**Reply by cwendt94** [reply]
The latest package version has the initial baseball implementation v0.14.0! I have not added the documentation yet but if you want to take a look at the code here is the League class https://github.com/cwendt94/espn-api/blob/master/espn_api/baseball/league.py

**Reply by tomrulz** [reply]
This is awesome. Thank you so much for getting this updated so quickly. I am going to try and attack it this week.
